web/test/e2e/docunit.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { test, expect } from "@playwright/test"
+
+test.describe("generate and delete doc units", () => {
+  let docUnitId: string
+
+  test("generate doc unit", async ({ page }) => {
+    await page.goto("/")
+
+    const newDocUnitButton = await page.waitForSelector(
+      "button >> text=Neue Dokumentationseinheit"
+    )
+    newDocUnitButton.click()
+
+    await page.waitForSelector("text=Rechtsprechung")
+    docUnitId = await page.locator(".panel-id").innerText()
+  })
+
+  test("upload docx", async ({ page }) => {
+    await page.goto("/")
+
+    const selectDocUnit = await page.waitForSelector(
+      `tr td:nth-child(1) a[href*="/rechtsprechung/${docUnitId}/dokumente"]`
+    )
+    selectDocUnit.click()
+
+    const [fileChooser] = await Promise.all([
+      page.waitForEvent("filechooser"),
+      page.locator("text=Festplatte durchsuchen").click(),
+    ])
+    await fileChooser.setFiles("./test/e2e/sample.docx")
+
+    await page.waitForSelector(
+      ".fileviewer-info-panel-value >> text=sample.docx"
+    )
+  })
+
+  test("delete docx", async ({ page }) => {
```

Comment:
Just a minor thing: maybe we should call it `delete original file` as we will support more file formats later on?

Reply:
done